src/services/entry-service.ts

```diff
@@ -88,19 +88,24 @@ class EntryService {
   }
 }
 
-const getDurationHumanizer = () => {
-  try {
-    const supportedLanguages = humanizeDuration.getSupportedLanguages();
-    const language = navigator.language;
-    if (supportedLanguages.includes(language)) {
-      return humanizeDuration.humanizer({language: language});
-    } else if (navigator.languages) {
-      const navigatorLanguages = navigator.languages.filter(language => supportedLanguages.includes(language));
-      return humanizeDuration.humanizer({language: navigatorLanguages[0]});
+const getDurationHumanizer = ({locale}: any) => {
+  const languages = ['en'];
+  if (locale) {
+    if (locale.match('_')) {
+      languages.unshift(locale.toLowerCase().split('_')[0]);
     }
-  } catch (e: any) {
-    return null;
+    languages.unshift(locale.toLowerCase());
   }
+
+  for (const language of languages) {
+    try {
+      if (humanizeDuration.getSupportedLanguages().includes(language)) {
```

Comment:
getSupportedLanguages should be lowercase no?

Reply:
yes

Reply:
not exactly
our languages are all in lowercase but they use zh_CN and zh_TW instead of zh_cn and zh_tw
I think using a Map might be the right solution

Reply:
so why not just lowercase?

Reply:
humanizeDuration.humanizer({ language: 'zh_cn' }) would throw an exception because it's not in their list of languages